packages/trading-widget/src/core-kit/hooks/trading/synthetix-v3/use-synthetix-v3-oracles-update.ts

```diff
@@ -58,8 +67,10 @@ export const useSynthetixV3OraclesUpdate = ({
     })
 
     try {
+      const oraclesUpdateTransactionData =
+        makeTrustedForwarderMulticall(prependedTxns)
       return sendTransaction?.({
-        to: oraclesUpdateTransactionData.to ?? AddressZero,
+        to: oraclesUpdateTransactionData.to,
         data: oraclesUpdateTransactionData.data,
         value: oraclesUpdateTransactionData.value,
       })
```

Comment:
I guess there is no need to override `from`, since it's connected to the wallet and it will be the connected wallet address?

Reply:
Yes, we are sending oracles update transaction using useSendTransaction wagmi hook, and it already has configured from property.